easybuild/easyconfigs/f/fenics-basix-py/fenics-basix-py-0.9.0-gfbf-2023b.eb

```
@@ -0,0 +1,36 @@
+# TODO: In Spack the package name is fenics-basix-py, but then the sanity checks
+# try to work with `fenics_basix_py`.
+name = 'basix'
```

```suggestion
name = 'fenics-basix-py'
```

With the Python package correctly detected, we can now use the name used for the folder 😄